packages/client/modules/userDashboard/components/OrgTeams/OrgTeams.tsx

```diff
@@ -31,23 +26,46 @@ const OrgTeams = (props: Props) => {
     `,
     organizationRef
   )
+  const {
+    open: openAddTeamDialog,
+    close: closeAddTeamDialog,
+    isOpen: isAddTeamDialogOpened
+  } = useDialogState()
+
   const {allTeams, isBillingLeader} = organization
   if (!isBillingLeader) return null
+
   return (
-    <>
-      <h1>{'Teams'}</h1>
-      <StyledPanel>
-        <Row>
-          <div className='flex w-full justify-between px-4'>
-            <div className='flex items-center font-bold'>Team Name</div>
-            <div className='flex items-center font-bold'>Lead</div>
+    <div className='max-w-4xl pb-4'>
+      <div className='flex items-center justify-center py-1'>
+        <h1 className='flex-1 text-2xl font-semibold leading-7'>Teams</h1>
+        <div className='ml-auto'>
+          <Button
+            variant='secondary'
+            shape='pill'
+            className='text-md w-32 py-2'
+            onClick={openAddTeamDialog}
+          >
+            Add team
+          </Button>
+        </div>
+      </div>
+
+      <div className='divide-y divide-slate-300 overflow-hidden rounded border border-slate-300 bg-white'>
+        <div className='bg-slate-100 px-4 py-2'>
+          <div className='flex w-full justify-between'>
+            <div className='flex items-center font-bold'>{allTeams.length} Teams</div>
```

Comment:
+1 if there's only one team, the copy should be "Team" rather than "Teams". We can use the plural helper function for this: https://github.com/ParabolInc/parabol/blob/8c68bf771a350b31c7948e6bd11c5c110e30c977/packages/client/utils/plural.ts

Reply:
^ this is from a review in progress, not complete

Reply:
Done!